app/services/control_rate_service.rb

```diff
@@ -10,60 +10,60 @@ def initialize(region, periods:)
     @region = region
     @facilities = region.facilities
     # Normalize between a single period and range of periods
-    @periods = if !periods.is_a?(Range)
+    @report_range = if !periods.is_a?(Range)
```

Comment:
Rename here for clarity, as there are two ranges in play from this layer down - the "report range" clients ask for, and the "full_data_range" that exists for a Region. The full data region is the entire set of data from the first registration for a Region to the current date.